src/SharedType.elm

```diff
@@ -8,5 +8,10 @@ type RepeatType
 
 
 type Tabs
-    = Kodi
+    = Kodi KodiTabs
     | Local
+
+
+type KodiTabs
+    = AudTab
```

Comment:
I don't like abbreviating this, or at all.

Reply:
To keep it short and simple to understand, I abbreviated the type. So, should the types be renamed AudioTab and VideoTab? It would be great to have your suggestions on this.